docs/src/ReactPlayground.js

```diff
@@ -46,9 +46,11 @@ const Portal = require('../../src/Portal');
 const ProgressBar = require('../../src/ProgressBar');
 const Row = require('../../src/Row');
 const SplitButton = require('../../src/SplitButton');
+const Tab = require('../../src/Tab');
 const TabbedArea = require('../../src/TabbedArea');
```

Comment:
These are set up as globals for the benefit of the `eval` below for the compiled code. Since you've replaced all the `TabbedArea` and `TabPane` usages in the examples, we should delete these old imports as well.